AppCenterReactNativeShared/android/build.gradle

```diff
@@ -28,12 +28,12 @@ allprojects {
     def ext = rootProject.ext
 
     android {
-        compileSdkVersion 27
-        buildToolsVersion '27.0.3'
+        compileSdkVersion 29
+        buildToolsVersion '29.0.2'
 
         defaultConfig {
-            minSdkVersion 16
-            targetSdkVersion 26
+            minSdkVersion 21
+            targetSdkVersion 29
```

Comment:
30?

Reply:
The target SDK version is 29 as it is currently used for newly created RN Android apps.